### PR TITLE
FLOE-532: Add news item for CISL Project Participation

### PR DESCRIPTION
**Pull request by waharnum**
Basic content for news item on the IDRC's partnership on the CISL project (http://www.cast.org/whats-new/news/2017/center-for-inclusive-software-for-learning.html).

### CODE REVIEW

File: news/2017-12-11-cast-cisl.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+                <p>
+                    We're pleased to announce that tools and knowledge from FLOE work on inclusive open education will be a part of to the newly-announced <a href="http://www.cast.org/whats-new/news/2017/center-for-inclusive-software-for-learning.html#.Wi7t9xNSwUE">Center on Inclusive Software for Learning (CISL)</a> as part of the <a href="https://idrc.ocadu.ca/">Inclusive Design Research Centre</a>'s partnership on the project.
+                </p>
+                <p>
+                    Led by <a href="http://www.cast.org/">CAST</a> and funded by the the U.S. Department of Education's <a href="https://www2.ed.gov/about/offices/list/osers/osep/index.html">Office of Special Education Programs</a>, CISL will support the development of an open-source software suite, industry guidelines and suporting research to ensure K-12 students get engaging personalized learning through Open Education Resources.
+                </p>
+                <p>
+                    We look forward to working with CAST and project partner <a href="https://www.sri.com/">SRI International</a> on advancing the state of the art in inclusive open education with FLOE work!
```
Comment (jobara) [inline news/2017-12-11-cast-cisl.html]: Would it still make sense if we removed the "work" from the end of the sentence. I'm thinking it's enough to just say FLOE. What do you think?

File: news/index.html [hunk 2]
```diff
@@ -83,6 +83,19 @@ <h1><a rel="home" title="Home" href="../index.html" class="floe-header-logo">flo
...
+                    <p>
+                        We're pleased to announce that tools and knowledge from FLOE work on inclusive open education will be a part of to the newly-announced <a href="http://www.cast.org/whats-new/news/2017/center-for-inclusive-software-for-learning.html#.Wi7t9xNSwUE">Center on Inclusive Software for Learning (CISL)</a> as part of the <a href="https://idrc.ocadu.ca/">Inclusive Design Research Centre</a>'s partnership on the project.
+                    </p>
+                        Led by <a href="http://www.cast.org/">CAST</a> and funded by the the U.S. Department of Education's <a href="https://www2.ed.gov/about/offices/list/osers/osep/index.html">Office of Special Education Programs</a>, CISL will support the development of an open-source software suite, industry guidelines and suporting research to ensure K-12 students get engaging personalized learning through Open Education Resources.
+                    </p>
+                    <p>
+                        We look forward to working with CAST and project partner <a href="https://www.sri.com/">SRI International</a> on advancing the state of the art in inclusive open education with FLOE work!
```
Comment (jobara) [inline news/index.html]: Same as previous comment about the word "work" at the end of the sentence.

Comment (waharnum) [reply news/index.html]: The text changes work for me, will do.

The CNAME file removal didn't originate in this PR but I think it should be done at some point - the site is no longer hosted on Github pages, making it surplus. I reverted it for this PR.

**Comment by jobara** [comment]
Merged at ba1a0ed2260808f2efe99c094938b4db6a9a0851

**Pull request merged**